src/driver/test/BigUpdate.yml

```diff
@@ -8,6 +8,9 @@
 
 SchemaVersion: 2018-07-01
 
+Pool:
+  QueryOptions:
+    maxPoolSize: 1000
```

Comment:
Oh, thank you. Could you open a ticket or PR against DSI to remove this setting from test_control.big_update.yml? 
This line: https://github.com/10gen/dsi/blob/f8514c63556b3071b68d5f3b21a4b686c6916a74/configurations/test_control/test_control.big_update.yml#L7

For my education, if set in both places, which one wins? They're the same right now so its moot, but what if they weren't?

Reply:
I believe the config wins right now. I should really put up a PR to make that explicitly clear. Of course, I'll PR this into DSI!